### PR TITLE
Adds documentation for deploying Aqueduct with Helm chart

### PR DESCRIPTION
**Pull request by cw75**

### CODE REVIEW

File: installation-and-deployment/deploying-with-helm.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Deploying on Kubernetes with Helm Chart
+
+To deploy Aqueduct on a Kubernetes cluster, you can use the helm chart we've created. Please follow the steps below.
```
Comment (vsreekanti) [inline installation-and-deployment/deploying-with-helm.md:3]: nit: Helm should be capitalized.

File: installation-and-deployment/deploying-with-helm.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+### Prerequisites
+
+* Follow the instruction [here](https://helm.sh/docs/intro/install/) to install Helm on your local machine.
+* You need to have access to an existing Kubernetes cluster. This means your local machine needs to have a kubeconfig file (typically located in `~/.kube/config`) that points to your cluster.
```
Comment (vsreekanti) [inline installation-and-deployment/deploying-with-helm.md:8]: this prereq should come first

"that points to the cluster you want to run Aqueduct on"

File: installation-and-deployment/deploying-with-helm.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+# Deploying on Kubernetes with Helm Chart
+
+To deploy Aqueduct on a Kubernetes cluster, you can use the helm chart we've created. Please follow the steps below.
+
+### Prerequisites
+
+* Follow the instruction [here](https://helm.sh/docs/intro/install/) to install Helm on your local machine.
```
Comment (vsreekanti) [inline installation-and-deployment/deploying-with-helm.md:7]: typo: "instructions"

File: installation-and-deployment/deploying-with-helm.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+   ```
+
+4. Install the Helm chart.
+   ```
+   helm install aqueduct/aqueduct --generate-name
+   ```
+
+### Customizing Install Options
+
+The installation command above deploys the latest version of Aqueduct in the default namespace with the server running with Python 3.8. The name of the deployment is generated with the `--generate-name` flag. You can customize all of these by running the following:
```
Comment (vsreekanti) [inline installation-and-deployment/deploying-with-helm.md:35]: This is a little confusing right now. Let's say that by default, we select Python 3.8.

File: installation-and-deployment/deploying-with-helm.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+   ```
+   helm install <DEPLOYMENT_NAME> aqueduct/aqueduct --namespace <NAMESPACE> --set image.repository=<REPO_NAME> --version <VERSION_NUMBER>
```
Comment (vsreekanti) [inline installation-and-deployment/deploying-with-helm.md:37]: Then let's explain what each of these do below.

File: installation-and-deployment/deploying-with-helm.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+   ```
+
+Supported image repositories are:
```
Comment (vsreekanti) [inline installation-and-deployment/deploying-with-helm.md:40]: Then explain this is how you configure Python versions.

**Review by vsreekanti — APPROVED** [review]
This looks good! A few nits inline to improve clarity.

**Comment by cw75** [comment]
@vsreekanti Does this version address your last three comments about customization?

**Comment by vsreekanti** [comment]
yep!

**Pull request merged**